src/react/components/pages/homepage/homePage.tsx

```diff
@@ -150,4 +189,16 @@ export default class HomePage extends React.Component<IHomepageProps> {
             throw new AppError(ErrorCode.ProjectDeleteError, "Error deleting project file");
         }
     }
+
+    private convertProject = async (project: IFileInfo) => {
+        const importService = new ImportService();
+        let projectJson;
+        try {
+            projectJson = await importService.convertV1(project);
```

Comment:
Instead of calling out `v1` in the method name I would just call it convert and leave it up the the import service to do the required transformations / version detection.